ote/src/cljs/ote/style/buttons.cljs

```diff
@@ -16,23 +21,38 @@
    :background-color colors/primary
    :transform "scale(0.98)"})
 
+(def negative-btn-hover-focus
+  {:text-decoration "none"
+   :box-shadow "0 4px 4px rgba(0,0,0,0.2)"
+   :background-color colors/negative-button-hover
+   :transform "scale(0.98)"})
+
 (def button-common
   {:padding "20px"
-   :height "60px"                                           ;;Hard coded so the height doesn't change with transitions
+   :height "60px"                                           ;; Hard coded so the height doesn't change with transitions
+   :min-width "4rem"
    :display "inline-block"
+   :justify-content "center"
```

Comment:
This only works with display "flex" elements. Should use text-align "center"

Reply:
True, will change. Thanks!